Server/mods/deathmatch/logic/CPlayer.h

```diff
@@ -465,4 +468,6 @@ class CPlayer final : public CPed, public CClient
 
     ushort  m_usPrevDimension;
     SString m_strQuitReasonForLog;
+
+    bool m_bHasPlayerSpawned;
```

Comment:
There already exists field CPed::m_bSpawned on server side. I know it behaves a bit different, but maybe there could be a way to avoid that collision. Because two fields with almost the same name may be confusing.

Reply:
> There already exists field CPed::m_bSpawned on server side. I know it behaves a bit different, but maybe there could be a way to avoid that collision. Because two fields with almost the same name may be confusing.

`m_bHasPlayerSpawned` will change to `false` when player dies. If I were to make these 2 variables into 1, I'd need to make a complete overhaul of the spawn system